style.css

```diff
@@ -1729,3 +1786,1422 @@ ul li {
   text-align: center;
   letter-spacing: 0.03em;
 }
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+  }
+
+  body {
+    position: relative;
+    box-sizing: border-box;
+    width: 1330px;
+    height: 125px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    background: #f4f5f7;
+    justify-content: center;
+    align-content: center;
+    margin: 0;
+  }
+
+  .container {
+    position: relative;
+    box-sizing: border-box;
+    width: 1350px;
+    height: 5000px;
+  }
+
```

Comment:
- [ ] Please from the project requirement, you should make sure that your project is responsive on the desktop version, but as you can see from the video below, your project has a horizontal scroll when adjusting the width of the desktop size. :blush:  I think you should adjust the style of the desktop version to make it responsive.

https://user-images.githubusercontent.com/92303891/206880238-a7db9031-b297-461a-94fa-dabdaf97cf6a.mp4